server/server.py

```diff
@@ -2,18 +2,22 @@
 from flask import Flask
 from slackeventsapi import SlackEventAdapter
 from command.core import Core
+import logging
 
 app = Flask(__name__)
 core = Core()
+logging.basicConfig(format='%(asctime)s - %(levelname)s : %(message)s',
+                    level=logging.INFO)
 
 
 @app.route('/')
 def check():
     """Display a Rocket status image."""
+    logging.info('ROCKET2 IS RUNNING')
```

Comment:
This function gets called every time the server serves a request to `/`, so I think this log message could be a bit more informative (e.g. `"Served 'check()'"` or `"Served route '/'"`), and the general "Rocket 2 startup log" should be placed outside (e.g. right below where logging is configured).

Also, just a nit, but I would prefer not to have all-caps logs - if all the logs are all-caps it'll be much harder to read in a console when debugging.